addons/website/static/src/snippets/s_dynamic_filter_carousel/options.js

```diff
@@ -0,0 +1,101 @@
+odoo.define('website.s_dynamic_filter_carousel_options', function (require) {
+'use strict';
+
+const options = require('web_editor.snippets.options');
+
+const SelectUserValueWidget = options.userValueWidgetsRegistry['we-select'];
+
+const DynamicFilterUserValueWidget = SelectUserValueWidget.extend({
```

Comment:
I don't follow the new 'option' of the left column, maybe @qsm-odoo can confirm that it is the correct way to have a dropdown filled from rpc content ?

Reply:
Indeed this is not the correct way ;) "UserValueWidget" are generic widgets which are meant to be reusable by any option (colorpicker, select, inputs, etc). This one is really specific. There are lots of room for improvement to share the code we have for "dynamic select widgets", but right now the correct way is using the `_renderCustomXML` (or `_renderCustomWidgets`) method. There are many examples with `_renderCustomXML` (none with `_renderCustomWidgets`...) :wink: